README.md

```diff
@@ -29,11 +29,8 @@ overridden by build and release pipelines.
 | APPLICATIONRESPONSE_QUEUE_ADDRESS               | Azure Service Bus message response queue name, e.g. `application-response`                            |
```

Comment:
General comment - code coverage is down and according to SonarCloud it will be <90% after merge

https://sonarcloud.io/summary/new_code?id=ffc-ahwr-application&pullRequest=118

Reply:
over 90% now :)